jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/CheckpointManager.scala

```diff
@@ -95,22 +97,22 @@ private[spark] class CheckpointManager(sc: SparkContext, checkpointPath: String)
   }
 
   /**
-    * Calculate a list of checkpoint rounds to save checkpoints based on the savingFreq and
-    * total number of rounds for the training. Concretely, the saving rounds start with
-    * prevRounds + savingFreq, and increase by savingFreq in each step until it reaches total
-    * number of rounds. If savingFreq is 0, the checkpoint will be disabled and the method
-    * returns Seq(round)
+    * Calculate a list of checkpoint rounds to save checkpoints based on the checkpointInterval
+    * and total number of rounds for the training. Concretely, the checkpoint rounds start with
+    * prevRounds + checkpointInterval, and increase by checkpointInterval in each step until it
+    * reaches total number of rounds. If checkpointInterval is 0, the checkpoint will be disabled
+    * and the method returns Seq(round)
     *
-    * @param savingFreq the increase on rounds during each step of training
+    * @param checkpointInterval the interval to save checkpoint to checkpointPath
     * @param round the total number of rounds for the training
     * @return a seq of integers, each represent the index of round to save the checkpoints
     */
-  private[spark] def getSavingRounds(savingFreq: Int, round: Int): Seq[Int] = {
-    if (checkpointPath.nonEmpty && savingFreq > 0) {
+  private[spark] def getCheckpointRounds(checkpointInterval: Int, round: Int): Seq[Int] = {
```

Comment:
Would you mind using Duration here or TimeUnit here? Neither the type nor the docs explain what the interval really is.

Reply:
Sorry for the confusion. It's the number of two iterations between two checkpoints instead of some kind of duration, similar to checkpointInterval in LDA in Spark.

Updated the scaladoc.